src/target/adiv5.c

```diff
@@ -810,16 +810,12 @@ void adiv5_dp_init(ADIv5_DP_t *dp, const uint32_t idcode)
 	/* This AP reset logic is described in ADIv5, but fails to work
 	 * correctly on STM32.	CDBGRSTACK is never asserted, and we
 	 * just wait forever.  This scenario is described in B2.4.1
-	 * so we have a timeout mechanism in addition to the sensing one.
-	 *
-	 * Write request for debug reset */
+	 * so we have a timeout mechanism in addition to the sensing one. */
+	platform_timeout_set(&timeout, 201);
```

Comment:
Did you mean to use 201 as the timeout in ms here? Would have expected 200 as the value for 10 loops of the loop below

Reply:
Changed value to 200 😄